docs/content/start/telemetry.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Collecting telemetry
+---
+
+
+Telemetry is an essential part of maintaining the health of modern applications. Fluid Framework provides a way to plug in your own logic to handle telemetry events sent by Fluid. This enables you to integrate the Fluid telemetry along with your other telemetry, and route the event data in whatever way you need.
+
+## ITelemetryBaseLogger interface
+
+The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can be implemented and passed into the client's `createContainer()` and `getContainer()` methods in both `FrsContainerConfig` and `TinyliciousContainerConfig` type parameters. Both `createContainer()` and `getContainer()` methods will create an instance of a `Loader` class object, where the logger defined in the `[client]ContainerConfig` is passed in as an optional parameter, `ILoaderProps.logger`, in the `Loader` constructor argument.
```

Comment:
What do the words "type parametes" mean in the 2nd sentence?

Reply:
Ok I get it now.  I might just say:

```suggestion
The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can be implemented and passed into the client's `createContainer()` and `getContainer()` methods via the config parameter, - Both `FrsContainerConfig` and `TinyliciousContainerConfig` have a member `logger?: ITelemetryBaseLogger` . Both `createContainer()` and `getContainer()` methods will create an instance of a `Loader` class object, where the logger defined in the `[client]ContainerConfig` is passed in as an optional parameter, `ILoaderProps.logger`, in the `Loader` constructor argument.
```

Reply:
yeah I think this sums it up better. 😀